config/initializers/session_store.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Rails.application.config.session_store :active_record_store, key: '_iria_next_session'
```

Comment:
**nitpick (blocking):** can you change this to `_irida_next_session`

Reply:
Fixed in b39da32